k8s/helm/yona-server/requirements.yaml

```diff
@@ -4,5 +4,5 @@ dependencies:
     repository:  "https://kubernetes-charts.storage.googleapis.com"
   - name: ldap
     version: 0.1.0
-    repository:  "file://../ldap"
+    repository:  "https://yonadev.github.io/helm-charts"
```

Comment:
This gave a merge conflict with my PR due to "condition: support_infrastructure.enabled" being added.  Might merge fine at master.